src/solver.cpp

```diff
@@ -330,6 +330,11 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
     RegisterWithSolver(
         registry, ++id, ConvAsmImplicitGemmV4R1DynamicWrw{}, miopenConvolutionAlgoImplicitGEMM);
 
+    RegisterWithSolver(registry,
```

Comment:
A new solver should always be added to the end of the list, so it doesn't change the solver-id of existing solvers

Reply:
Yes, right!